.github/CODEOWNERS

```diff
@@ -1,4 +1,9 @@
-* @Xuanwo @PsiACE @ClSlaid @sundy-li
+* @Xuanwo
```

Comment:
I am glad to receive the message from opendal, maybe I can stay here.

Reply:
Thanks!